framework-cloud/cloud-deployment-plugin/pom.xml

```diff
@@ -7,7 +7,7 @@
   <parent>
     <groupId>org.kie.cloud</groupId>
     <artifactId>framework-cloud</artifactId>
-    <version>7.4.1-SNAPSHOT</version>
+    <version>7.5.0-SNAPSHOT</version>
```

Comment:
Is it really necessary to have this in all the poms? Wouldn't using `${project.parent.version}` be better?

Reply:
@jhrcek Unfortunately no, Maven isn't able to resolve expression in parent version.